### ISSUE TITLE
highlighting off while stepping

### ISSUE THREAD

**Issue by ericwinger** [ericwinger/Jade#75]
Rowan sha: ef4afb
Jadeite: 3.0.14

- Create new method 

```
RowanSample1>>streamArray
	| array ws |
	array := Array with: 'abc' with: 'def'. 
self halt. 
	ws := WriteStream on: String new. 
	1 to: array size do:[:idx |
			ws nextPutAll: (array at: idx).
			ws space.
	].
	^ws contents

```
- Turn on the new debugger ``RowanDebugger debuggerClass: RowanDebugger``
- Step until ``size`` is highlighted. 
- Step one more time. Note that highlighting is not on ``to:do:``

![image](https://user-images.githubusercontent.com/36212516/44356445-381e6100-a464-11e8-97ff-802658a6edd2.png)

**Comment by ericwinger** [comment]
In retrospect, this appears to be an extra step point rather than incorrect highlighting. 

Editing title

**Comment by jgfoster** [comment]
Yes, there are a number of issues with how the server reports step points. Unfortunately, they won't be fixed in 3.2.x. Fortunately things have improved in later versions.
